aes-gcm-siv/src/lib.rs

```diff
@@ -261,9 +260,13 @@ where
 
         // Zeroize all intermediate buffers
         // TODO(tarcieri): use `Zeroizing` when const generics land
-        mac_key.as_mut_slice().zeroize();
-        enc_key.as_mut_slice().zeroize();
-        block.as_mut_slice().zeroize();
+        #[cfg(feature = "zeroize")]
+        {
+            use zeroize::Zeroize;
+            mac_key.as_mut_slice().zeroize();
+            enc_key.as_mut_slice().zeroize();
+            block.as_mut_slice().zeroize();
```

Comment:
I am not sure we need these zeroizations of stack-based values since we generally do not deal with stack spilling.